spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala

```diff
@@ -39,6 +39,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
           .set("spark.kryoserializer.buffer.mb", "15")
           .set("spark.akka.frameSize", "30")
           .set("spark.default.parallelism", "10")
+          .set("spark.driver.allowMultipleContexts", "true")
```

Comment:
Hm... I think i want to understand this more.

I suspect the right thing to do is to understand why how our test setup is
flawed for 1.2  and fix context instantiation rather than put this property
in.

Could we please delay this until we move on to 1.2 at which point.

Otherwise i suppose i don't see a reason not to bump to 1.1.1 at this point.


On Tue, Jan 27, 2015 at 3:00 PM, Erlend Hamnaberg <notifications@github.com>
wrote:

> In
> spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala
> <https://github.com/apache/mahout/pull/71#discussion_r23651449>:
>
> > @@ -39,6 +39,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
> >            .set("spark.kryoserializer.buffer.mb", "15")
> >            .set("spark.akka.frameSize", "30")
> >            .set("spark.default.parallelism", "10")
> > +          .set("spark.driver.allowMultipleContexts", "true")
>
> This was done since the tests failed with an error message similar to:
> "Please enable this property to make tests work". Currently using my phone,
> so can't check exact error message. To reproduce: set spark version to
> 1.2.0 and run tests.
> … <#14b2d9fb32e77c9c_>
> On 27 Jan 2015 21:09, "Dmitriy Lyubimov" <notifications@github.com>
> wrote: In
> spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala
> <https://github.com/apache/mahout/pull/71#discussion_r23638378>: > @@
> -39,6 +39,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite
> with LoggerConfigurat > .set("spark.kryoserializer.buffer.mb", "15") >
> .set("spark.akka.frameSize", "30") > .set("spark.default.parallelism",
> "10") > + .set("spark.driver.allowMultipleContexts", "true") @hamnis
> <https://github.com/hamnis> <https://github.com/hamnis>, could you please
> explain what is significance of this parameter -- "allowMultipleContexts"
> in 1.2.0 so that tests won't run? As far as i remember, current head is
> using only one context (per suite).so next suite should run another
> context, but by that time the previous one should already be gone. — Reply
> to this email directly or view it on GitHub <
> https://github.com/apache/mahout/pull/71/files#r23638378>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/71/files#r23651449>.
>